extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -1411,17 +1424,7 @@ private boolean isTaskCurrent(int taskGroupId, String taskId)
       }
     }
 
-    return Futures.transform(
-        Futures.successfulAsList(futures), new Function<List<Void>, Void>()
-        {
-          @Nullable
-          @Override
-          public Void apply(@Nullable List<Void> input)
-          {
-            return null;
-          }
-        }, workerExec
-    );
+    return asVoidFuture(Futures.successfulAsList(futures));
```

Comment:
Maybe `stopTaskInGroup()` could return `ListenableFuture<?>` and this wrapping `asVoidFuture` is not needed?